types/vfile/index.d.ts

```diff
@@ -0,0 +1,72 @@
+// Type definitions for vfile 2.2
+// Project: https://github.com/vfile/vfile#readme
+// Definitions by: Junyoung Choi <https://github.com/rokt33r>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+declare namespace vfile {
+    type Contents = string | Buffer;
+
+    interface Position {
+        line: number;
+        column: number;
+    }
+
+    interface Location {
```

Comment:
Use [`Position`](https://github.com/syntax-tree/unist#position) as a name for this interface.